pkg/cli/cmd/group/groupswitch/switch_test.go

```diff
@@ -156,6 +158,7 @@ func Test_Run(t *testing.T) {
 				ConnectionFactory:    &connections.MockFactory{ApplicationsManagementClient: appManagementClient},
 				Workspace:            workspace,
 				UCPResourceGroupName: "a",
+				Output:               outputSink,
```

Comment:
Can you add an assert below about the output?

Here's an example you can borrow from: https://github.com/radius-project/radius/blob/main/pkg/cli/cmd/app/delete/delete_test.go#L280